src/content/docs/kubernetes-pixie/k8s-otel/intro.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: 'Introduction to OpenTelemetry for Kubernetes'
+tags:
+    - Kubernetes integration
+    - OpenTelemetry
+    - New Relic OpenTelemetry
+    - OpenTelemetry Collector
+    - OpenTelemetry Kubernetes
+    - OpenTelemetry Helm chart
+    - New Relic OpenTelemetry installation
+metaDescription: "Learn how to monitor your Kubernetes Cluster using OpenTelemetry"
+freshnessValidatedDate: 2024-07-23
+redirects:
+    - /docs/kubernetes-pixie/kubernetes-integration/installation/k8s-otel/
+---
+
+
+OpenTelemetry observability for Kubernetes provides complete, open-source setup paired with a top-notch Kubernetes UI that is already compatible with our proprietary Kubernetes instrumentation. Our Kubernetes UIs are designed to be provider agnostic, allowing you to select either OpenTelemetry or New Relic instrumentation based on your needs.
```

Comment:
Shoud we add in "proprietary Kubernetes intrumentation" to our main K8s install document?